src/components/EmojiPicker/EmojiPicker.vue

```diff
@@ -0,0 +1,236 @@
+	### General description
+
+	This component allows the user to pick an emoji.
+
+	### Usage
+
+	* Listen to the select event and pass in an HTML element that will be treated as a trigger:
+
+	```vue
+	<template>
+		<div>
+			<EmojiPicker @select="log">
+				<button> Click Me </button>
+			</EmojiPicker>
+		</div>
+	</template>
+	<script>
+		export default {
+			methods: {
+				log(e) {
+					console.debug(e)
+				},
+			},
+		}
+	</script>
+	```
+
+	* Showing a preview and keeping it open after a user selected an emoji
+
+	```vue
+	<template>
+		<div>
+			<EmojiPicker
+				:close-on-select="false"
+				:show-preview="true"
+				@select="log">
+				<button> Click Me </button>
+			</EmojiPicker>
+		</div>
+	</template>
+	<script>
+		export default {
+			methods: {
+				log(e) {
+					console.debug(e)
+				},
+			},
+		}
+	</script>
+	```
+
+</docs>
+
+<template>
+	<Popover
+		:open.sync="open"
+		popover-inner-class="popover-emoji-picker-inner">
+		<template #trigger>
+			<slot />
+		</template>
+		<Picker
+			:auto-focus="true"
+			color="var(--color-primary)"
+			:data="emojiIndex"
+			:emoji="emoji"
+			:i18n="i18n"
+			:native="native"
+			:show-preview="showPreview"
+			:title="title"
+			v-bind="$attrs"
+			@select="select" />
+	</Popover>
+</template>
+
+<script>
+import { Picker, EmojiIndex } from 'emoji-mart-vue-fast'
+import data from 'emoji-mart-vue-fast/data/all.json'
+import Popover from '../Popover'
+import { t } from '../../l10n'
+
+export default {
+	name: 'EmojiPicker',
+	components: {
+		Picker,
+		Popover,
+	},
+	props: {
+		/**
+		 * The emoji-set
+		 */
+		activeSet: {
+			type: String,
+			default: 'native',
+		},
+		/**
+		 * Show preview section when hovering emoji
+		 */
+		showPreview: {
+			type: Boolean,
+			default: false,
+		},
+		/**
+		 * The fallback emoji in the preview section
+		 */
+		emoji: {
+			type: String,
+			default: 'grinning',
+		},
+		/**
+		 * The fallback text in the preview section
+		 */
+		title: {
+			type: String,
+			default: t('Pick an emoji'),
+		},
+		/**
+		 * Whether to close the emoji picker after picking one
+		 */
+		closeOnSelect: {
+			type: Boolean,
+			default: true,
+		},
+	},
+	data() {
+		return {
+			emojiIndex: new EmojiIndex(data),
+			i18n: {
+				search: t('Search'),
+				notfound: t('No Emoji Found'),
+				categories: {
+					search: t('Search Results'),
+					recent: t('Frequently Used'),
+					smileys: t('Smileys & Emotion'),
+					people: t('People & Body'),
+					nature: t('Animals & Nature'),
+					foods: t('Food & Drink'),
+					activity: t('Activities'),
+					places: t('Travel & Places'),
+					objects: t('Objects'),
+					symbols: t('Symbols'),
+					flags: t('Flags'),
+					custom: t('Custom'),
+				},
+			},
+			open: false,
+		}
+	},
+	computed: {
+		native() {
+			return this.activeSet === 'native'
+		},
+	},
+	methods: {
+		select(emojiObject) {
+			/**
+			 * Emits a string containing the emoji e.g. '👩🏿‍💻'
+			 */
+			this.$emit('select', emojiObject.native)
```

Comment:
Maybe also `update:select`? :)
To use with sync modifier?

Reply:
I'm not entirely sure I understand how a sync modifier is helpful here, since there is no property which is being updated?

There is no selected emoji property you are providing to this component.

Reply:
Ah, sorry, I meant `update:emoji`
I misunderstood its use I guess :)